### PR TITLE
Features/ingestform

### LINKED ISSUE #3: Ingest form
As an administrator I want to be able to upload a single file and associate RDF metadata with the object so that it can be displayed to users.

### PR DESCRIPTION
**Pull request by zhang4952**
fixes #3

### CODE REVIEW

File: app/presenters/my_generic_file_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# app/presenters/my_generic_file_presenter.rb
+class MyGenericFilePresenter < Sufia::GenericFilePresenter
```
Comment (terrellt) [inline app/presenters/my_generic_file_presenter.rb]: Can we rename this to FilePresenter?

File: app/forms/my_file_edit_form.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# app/forms/my_file_edit_form.rb
+class MyFileEditForm < MyGenericFilePresenter
```
Comment (terrellt) [inline app/forms/my_file_edit_form.rb]: Rename to FileEditForm?

File: app/forms/my_batch_edit_form.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# app/forms/my_batch_edit_form.rb
+class MyBatchEditForm < MyFileEditForm
```
Comment (terrellt) [inline app/forms/my_batch_edit_form.rb]: Rename to BatchEditFOrm?

File: app/presenters/file_presenter.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# app/presenters/file_presenter.rb
+class FilePresenter < Sufia::GenericFilePresenter
+  self.terms = [:resource_type, :title, :creator, :contributor, :description,
```
Comment (terrellt) [inline app/presenters/file_presenter.rb]: Not all the properties in the model are here - is that on purpose?

**Comment by terrellt** [comment]
Branch needs rebased.

**Comment by straleyb** [comment]
Looks good. @luisgreg99?

**Comment by terrellt** [comment]
@zhang4952 Can you squash these commits? This is a lot of noise in the commit history.

**Comment by straleyb** [comment]
@zhang4952 @terrellt Is a generic file supposed to have an attribute called "type"?

**Comment by terrellt** [comment]
Can't have type, it's a reserved term.

**Comment by straleyb** [comment]
Looks like it needs to be taken out of the presenter then.

**Comment by luisgreg99** [comment]
@zhang4952 , I got an undefined attribute error after uploading a file as an admin. @straleyb suggested removing the "type" field in the FilePresenters class, which fixes the error. Is the "type" field required?

**Comment by luisgreg99** [comment]
In that case, we can rename it if it's a required field.

**Comment by luisgreg99** [comment]
@zhang4952 I tested the new change and it's working as expected, @terrellt any thoughts before merging to master?

**Pull request merged**